tests/ember-configuration.js

```diff
@@ -99,7 +99,7 @@
     registry.register('adapter:-json-api', DS.JSONAPIAdapter);
     registry.register('serializer:-json-api', DS.JSONAPISerializer);
 
-    registry.register('service:store', DS.Store);
```

Comment:
Maybe this one could have been kept ? I think it's better to register the factories and let the framework do the injection.